cmd/bosun/sched/check.go

```diff
@@ -140,6 +135,61 @@ func (s *Schedule) runHistory(r *RunHistory, ak models.AlertKey, event *models.E
 			}
 		}
 	}()
+	if incident != nil {
+		rt, err = data.GetRenderedTemplates(incident.Id)
+		if err != nil {
+			return
+		}
+		for i, action := range incident.Actions {
+			if action.Type == models.ActionDelayedClose && !(action.Fullfilled || action.Cancelled) {
+				if event.Status > incident.WorstStatus {
+					// If the lifetime severity of the incident has increased, cancel the delayed close
+					cerr := s.ActionByAlertKey("bosun", "cancelled delayed close due to severity increase", models.ActionCancelClose, nil, ak)
+					incident, err = data.GetIncidentState(incident.Id)
+					if cerr != nil {
```

Comment:
Are we ok logging and continuing on `cerr`?

If so, this logic is a bit off, and `err` is unchecked.

Perhaps:

```
cerr := s.ActionByAlertKey(...)
if cerr != nil{
  // slog, no need to refetch
}else{
  // refetch and check err
}
```